src/pages/VoterLeaderboard/LeaderboardTable/index.tsx

```diff
@@ -0,0 +1,136 @@
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import { Link } from 'react-router-dom';
+import { Typography } from '@mui/material';
+import { EllipseText, Table, TableProps } from 'components';
+import Path from 'constants/path';
+import { useTranslation } from 'translation';
+import { VoterAccount } from 'types';
+import { convertWeiToTokens, formatToReadablePercentage } from 'utilities';
+import { useStyles } from './styles';
+
+export interface ILeaderboardTableProps extends Pick<TableProps, 'getRowHref'> {
+  voterAccounts: VoterAccount[];
+  offset: number;
+  isFetching: boolean;
+}
+
+export const LeaderboardTable: React.FC<ILeaderboardTableProps> = ({
+  voterAccounts,
+  offset,
+  isFetching,
+}) => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+
+  const columns = useMemo(
+    () => [
+      { key: 'rank', label: t('voterLeaderboard.columns.rank'), orderable: false, align: 'left' },
+      {
+        key: 'votes',
+        label: t('voterLeaderboard.columns.votes'),
+        orderable: false,
+        align: 'right',
+      },
+      {
+        key: 'voteWeight',
+        label: t('voterLeaderboard.columns.voteWeight'),
+        orderable: false,
+        align: 'right',
+      },
+      {
+        key: 'proposalsVoted',
+        label: t('voterLeaderboard.columns.proposalsVoted'),
+        orderable: false,
+        align: 'right',
+      },
+    ],
+    [],
+  );
+
+  const cardColumns = useMemo(() => {
+    const newColumns = [...columns];
+    newColumns[2].align = 'center';
+    newColumns[3].align = 'left';
+    return newColumns;
+  }, [columns]);
```

Comment:
These aren't based on any state or props, so they can be declared outside the component

Reply:
They rely on the translation context though